sup3r/pipeline/forward_pass.py

```diff
@@ -311,6 +308,18 @@ def s_hr_slices(self):
                     self._s_hr_slices.append(hr_slice)
         return self._s_hr_slices
 
+    @property
+    def gids(self):
+        """Get gids for spatial coordinates. Used in final data collection."""
```

Comment:
Can you describe what this returns? It's a dictionary with keys=node index and values=arrays of spatial gids per node? Are the spatial gids the same as the gids in the final collected output h5? If not, we need to be very careful and maybe rename this.

Reply:
Yeah its a dictionary of unique indices for each lat/lon pair for forward pass output. The collected output will have a set of gids equal to the full set.

Reply:
Yeah just add this to a "returns" docstring. Are the gid's the same order as the flattened h5 output?

Reply:
That's a good point. They should be but maybe this could be related to the seam issue